users/README.md

```diff
@@ -34,6 +34,12 @@
 		"super_user": "boolean",
 		"in_discord": "boolean"
 	},
+    "archivedDetails": {
+    "archived_at": "String",
+    "archived_user_id": "String",
+    "super_user_id": "String",
```

Comment:
`updatedBy` sounds more better

Reply:
okay, updating this to `updated_by` so it will be consistent with other keys